command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Thu, Jul 28, 2016 at 12:13:35PM -0700, Mike Brown wrote:
> > @@ -0,0 +1,189 @@
> > +# Operations
> > +
> > +A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
>
> I think the purpose is to say if you have a compliant CLI it must
> adhere... But you could also have a different type of runtime
> interface such as a restful web interface.  We would just have to
> write up a spec for that if we wanted interop over such.

Until we have such an API as part of runtime-spec, I don't see how
you'd certify an HTTP-only runtime as compliant.